### PR TITLE
feat: added members page subnav popover

### PR DESCRIPTION
**Pull request by CzarekDryl**
Added subnav popver for members page

https://tw.auroracreation.com/app/tasks/15496547

image

### CODE REVIEW

File: src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.module.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.navigationButton {
+  @apply w-full flex items-center py-2 px-4 hover:bg-gray-50 hover:font-medium transition-all duration-normal rounded;
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.module.css:2]: Are you sure we should use font medium on hover here? It causes a shift of hovered elements.

File: src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+import Tooltip from '~shared/Extensions/Tooltip';
+import Icon from '~shared/Icon';
+import { useMembersSubNavigation } from './hooks';
+import styles from './SubNavigation.module.css';
+
+const SubNavigation = () => {
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.tsx:9]: Please, add the displayName value and use FC to create a React component as in other files.

File: src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+              <span className="text-3 underline">
+                {formatMessage({
+                  id: isCopyTriggered ? 'copiedColony' : 'copyColony',
+                })}
+              </span>
+            }
+            isSuccess={isCopyTriggered}
+          >
+            <Icon name="share-network" appearance={{ size: 'small' }} />
+            <span className="text-md ml-2">Invite members</span>
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.tsx:32]: Are these values shouldn't be used with translation?

File: src/components/v5/shared/BurgerMenu/BurgerMenu.tsx [hunk 4]
```diff
@@ -1,14 +1,15 @@
 import React, { FC } from 'react';
 
-import Icon from '~shared/Icon/Icon';
+import Icon from '~shared/Icon';
 import { BurgerMenuProps } from './types';
 
-const displayName = 'v5.shared.BurgerMenu.BurgerMenu';
+const displayName = 'v5.shared.BurgerMenu';
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/v5/shared/BurgerMenu/BurgerMenu.tsx:6]: const displayName = 'v5.BurgerMenu'

All components in shared directory don't have a 'shared' prefix in display name

File: src/components/v5/shared/BurgerMenu/BurgerMenu.tsx [hunk 5]
```diff
@@ -1,14 +1,15 @@
...
 
-const BurgerMenu: FC<BurgerMenuProps> = ({ isVertical }) => (
+const BurgerMenu: FC<BurgerMenuProps> = ({ isVertical, setTriggerRef }) => (
   <button
     type="button"
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/v5/shared/BurgerMenu/BurgerMenu.tsx:10]: If it's just a icon inside the button, we should add aria-label to them

File: src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.tsx (hunk 2 above)
Comment (CzarekDryl) [reply src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.tsx:9]: Added

File: src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.tsx (hunk 3 above)
Comment (CzarekDryl) [reply src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.tsx]: Translations added

File: src/components/v5/shared/BurgerMenu/BurgerMenu.tsx (hunk 4 above)
Comment (CzarekDryl) [reply src/components/v5/shared/BurgerMenu/BurgerMenu.tsx]: Fixed

File: src/components/v5/shared/BurgerMenu/BurgerMenu.tsx (hunk 5 above)
Comment (CzarekDryl) [reply src/components/v5/shared/BurgerMenu/BurgerMenu.tsx]: aria-label added

File: src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.module.css (hunk 1 above)
Comment (CzarekDryl) [reply src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.module.css:2]: It's added on designs, so I will leave it for now

**Review by KatarzynaStefanowiczPagepro — APPROVED** [review]

File: src/components/frame/v5/pages/MembersPage/MembersPage.tsx [hunk 6]
```diff
@@ -2,11 +2,41 @@ import React, { FC } from 'react';
...
-);
+const MembersPage: FC = () => {
+  const { getTooltipProps, setTooltipRef, setTriggerRef, visible } =
+    useMembersPage();
+
+  return (
+    <TwoColumns aside={<Navigation pageName="members" />}>
+      <div className="flex items-center justify-between">
+        <div className="flex items-center">
+          <h4 className="heading-4 mr-2">All members</h4>
```
Comment (krawczykkatarzyna) [inline src/components/frame/v5/pages/MembersPage/MembersPage.tsx:18]: add missing translaction

Comment (CzarekDryl) [reply src/components/frame/v5/pages/MembersPage/MembersPage.tsx]: Fixed

**Review by krawczykkatarzyna — APPROVED** [review]

**Pull request merged**